libflux/src/semantic/mod.rs

```diff
@@ -1,10 +1,21 @@
 mod analyze;
+pub use analyze::analyze;
+
 mod env;
 mod fresh;
 mod infer;
+
+// TODO(jsternberg): Once more work is done on the infer methods,
+// this should be removed and the warnings fixed.
+#[allow(warnings)]
```

Comment:
Can you create an issue to clean up these warnings?

Reply:
Sure.

Reply:
https://github.com/influxdata/flux/issues/2106